### DISCUSSION TITLE
MiXCR and Firewall SSL inspection

### DISCUSSION THREAD

**Discussion by savinilya** [milaboratory/mixcr#1716] [Q&A] [answered]
Hi! When i try to install new license to the MiXCR application in my HPC environment, SSL inspection from firewall breaks license checking with "License Error: Connection Error. License Manager thread died" error. It is possible to add our internal CA to MiXCR keychain or disable ssl checking?

**Comment by mizraelson** [comment, marked as answer]
Hi,
For running MiXCR in the environment with restricted internet access, please add the following MiXCR IPs to the whitelist in the firewall settings:

```
IPv4:
75.2.96.100
99.83.215.63

IPv6:
2600:9000:a403:55e8:f9c1:f443:773b:4192
2600:9000:a51b:2483:639d:d5a9:270f:6439
```

Let me know if it helps.
